cli/tests/pcluster/config/test_update_policy.py

```diff
@@ -1652,3 +1652,115 @@ def test_condition_checker_managed_fsx(
     assert_that(condition_checker_managed_fsx(change, patch)).is_equal_to(expected_subnet_updated)
     assert_that(fail_reason_managed_fsx(change, patch)).is_equal_to(expected_fail_reason)
     assert_that(actions_needed_managed_fsx(change, patch)).is_equal_to(expected_action_needed)
+
```

Comment:
Are we allowed to `add` or `remove` a `LoginNodes` section from a running cluster ?

If the answer is yes we need a test for this.

Reply:
I agree. I've checked how to do that and unfortunately it's not enough to add a test case in this unit test, but require a more complex unit test. i'll address this comment in a follow up PR.